### DISCUSSION TITLE
github.pullRequests stopped working

### DISCUSSION THREAD

**Discussion by staticdev** [community/community#25352] [Actions]
<p>I had a working script and it stopped working. I didn’t have any notice or warning of a change in Actions.</p>
<p>I was using:</p>
<pre><code class="lang-auto">jobs:
  automerge:
    name: Merge Dependabot pull request
    runs-on: ubuntu-latest
    if: github.base_ref == 'master' &amp;&amp; github.actor == 'dependabot[bot]'
    steps:
      - uses: actions/github-script@v2
        with:
          github-token: ${{secrets.GITHUB_TOKEN}}
          script: |
            github.pullRequests.merge({
              owner: context.repo.owner,
              repo: context.repo.repo,
              pull_number: context.payload.pull_request.number
            })
</code></pre>
<p>Now <a href="https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#github-context" rel="nofollow noopener">it seems by the documentation</a> github.pullRequests don’t exist anymore and I get the error: <code> Unhandled error: TypeError: Cannot read property 'merge' of undefined</code>.</p>
<p>Any hints here?</p>

**Comment by Yanjingzhu** [comment, marked as answer]
<p><a class="mention" href="/staticdev">@staticdev</a> Please use <code>github.pulls.merge</code> .<br>
In <code>actions/github-script@v2</code>,   <code>github</code>  object is a pre-authenticated <a href="https://github.com/octokit/rest.js" rel="nofollow noopener">octokit/rest.js</a> client.<br>
According to <a href="https://octokit.github.io/rest.js/v18" rel="nofollow noopener">https://octokit.github.io/rest.js/v18</a> , pull request merge function is this :<br>
</p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/0/03bf88e75d6cc1960e05dc5aea0fa382fefee976.png" data-download-href="/uploads/short-url/x9NdjlTlqImJN74JZaD0DwdbgO.png?dl=1" title="host local" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/24806478/181090576-90f452a7-7318-484c-808a-d2567d12e81c.png" alt="host local" data-base62-sha1="x9NdjlTlqImJN74JZaD0DwdbgO" width="690" height="383" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/0/03bf88e75d6cc1960e05dc5aea0fa382fefee976_2_690x383.png, https://aws1.discourse-cdn.com/github/optimized/2X/0/03bf88e75d6cc1960e05dc5aea0fa382fefee976_2_1035x574.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/0/03bf88e75d6cc1960e05dc5aea0fa382fefee976.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/0/03bf88e75d6cc1960e05dc5aea0fa382fefee976_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">host local</span><span class="informations">1138×633 22.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by staticdev** [comment]
<p>That worked as charm! Thanks a lot <a class="mention" href="/yanjingzhu">@yanjingzhu</a>!</p>
